test/minitest_ext.rb

```diff
@@ -43,8 +44,12 @@ def to_s # :nodoc:
 
     private
 
-    def stub_monorail_log_invocation
+    def stub_monorail_log_git_sha
       ShopifyCli::Git.stubs(:sha).returns("bb6f42193239a248f054e5019e469bc75f3adf1b")
     end
+
+    def stub_prompt_for_cli_updates
```

Comment:
Should we just stub out the whole `before_resolve` block in entrypoint? We shouldn't be passing through any of the update code.

Reply:
This is also a bigger task, that we need to stop using `run_cmd`

Reply:
Good question. Maybe I'll leave it here for now but we can address it as part of test stability.